docs/further_reading.dialog_element.md

```diff
@@ -3,19 +3,33 @@ title: Dialog element
 
-- Clicking the backdrop does not close the dialog on Chrome.
-- The native `::backdrop` only shows when programatically opening the dialog, not when using the `open` attribute.
+:::info  
+For extensive information about the `<dialog>` element, do read [Having an open dialog](https://www.scottohara.me/blog/2019/03/05/open-dialog.html) and more importantly [Use the dialog element (reasonably)](https://www.scottohara.me/blog/2023/01/26/use-the-dialog-element.html), from Scott O’Hara.  
+:::
+
+## Remaining caveats
+
+Here are some of the remaining issues with the HTML `<dialog>` element:
+
+- Clicking outside the dialog does not close the dialog in some browsers such as Chrome.
+- The `::backdrop` pseudo-element only shows when programatically opening the dialog, not when using the `open` attribute.
 - Default styles are left to the browsers’ discretion and can be inconsistent.
-- The [alert dialog pattern](advanced.alert_dialog.md) (`role="alertdialog"`) simply does not work with the dialog element.
-- It still requires JavaScript anyway, so it’s not even 100% HTML.
-- [Read more about the shortcoming of the dialog element by Scott O'hara](https://www.scottohara.me/blog/2019/03/05/open-dialog.html).
+- The [alert dialog pattern](advanced.alert_dialog.md) (`role="alertdialog"`) does not work with the `<dialog>` element.
+- Using the `<dialog>` element still requires JavaScript (unlike `<details>` for instance).
+
+:::note  
+If you are aware of a downside or pitfall of the `<dialog>` element that was not mentioned here, please kindly [open an issue](https://github.com/KittyGiraudel/a11y-dialog/issues) so we can keep this document relevant and informative.  
+:::
+
+## Why a11y-dialog
 
-Additionally, there are still reasons to use a11y-dialog (or a similar library):
+This brings us to the reasons why one may stick to using a11y-dialog (or a similar library):
 
-- Not wanting to deal with browsers inconsistencies. It would be a bad reason to use a JavaScript library **if** `<dialog>` worked without JavaScript, which it doesn’t. So if one wants a dialog that looks the same everywhere, then a library like a11y-dialog remains the way to go.
-- Having to support old browsers without the `<dialog>` element. I guess a polyfill might be better in a case like that, but also maybe not. Might as well use the same library for everything.
+- Not wanting to deal with browsers inconsistencies. It would be a bad reason to use a JavaScript library **if** the `<dialog>` element worked without JavaScript, which it doesn’t. So if one wants a dialog that looks the same everywhere, then a library like a11y-dialog remains a way to go.
+- Having to support old browsers without support for the `<dialog>` element. I guess a polyfill might be better in a case like that, but also maybe not. Note that a11y-dialog v8 is geared towards modern browsers though.
```

Comment:
```suggestion
- Needing to support browsers that don't support the `<dialog>` element. While polyfills exist (such as [Google's polyfill on GitHub](https://github.com/GoogleChrome/dialog-polyfill)), they are larger than a11y-dialog and provide *fewer* features with *more* code. a11y-dialog takes care to be small and efficient, with a focus on supporting modern browsers.
```

Reply:
If calling out the google polyfill, may be a good idea to specifically mention it does not have full support for all a11y features

Reply:
Good point; thank you.